frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -28,26 +28,32 @@ end
 -- Load the user dictionary suitable for the actual language
 -- if reload==true, force a reload
 -- Unload is done automatically when a new dictionary is loaded.
-function ReaderUserHyph:loadDictionary(name, reload)
+-- _second_try is reserved internally if the user dictionary gets scrubbed/sorted
+function ReaderUserHyph:loadDictionary(name, reload, _second_try)
```

Comment:
No need for this comment, and no need for the underscore. It's all internal :)
Just name it `no_scrubbing`, so it documents itself.